src/web/src/components/Banner.tsx

```diff
@@ -90,7 +89,11 @@ const useStyles = makeStyles((theme: Theme) =>
   })
 );
 
-export default function Banner() {
+type BannerProps = {
+  onVisibilityChange: (is: any) => void;
```

Comment:
A boolean would be better here, we tend to avoid `any` in TS.
And why do you  name the `onVisibilityChange` argument `is` and `event` in two places.

Reply:
@DukeManh thank you for the review!
Yes it used to be `=> boolean` but then it'll cause the typescript error when I do `onVisibilityChange={(e) => setBannerVisibility(e)` when I call `<Banner>`.  That's why I change it to `void` instead. 

Do you have suggestion how to solve this error?

Reply:
@yuanLeeMidori It looks like the return type `void` is correct. I was referring to the argument `is`. It should be a boolean according to your usage on line 164 and 166.
```suggestion
  onVisibilityChange: (event: boolean) => void;
```

Reply:
Got it! Thank you! I've pushed new commit with change :)